### PR TITLE
refactor(app): fix bad release branch merge

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

Was too eager in merging the release branch back into edge and broke CI. 

# Review requests

@vabruzzo for some reason the changes you made to `ConfigFormSubmitButton.tsx` didn't get tracked, can you verify this is what it should look like?

# Risk assessment

Low

### CODE REVIEW

File: app/src/organisms/ConfigurePipette/ConfigFormSubmitButton.tsx [hunk 1]
```diff
@@ -9,22 +9,23 @@ import {
...
-  const { disabled } = props
+  const { disabled, formId } = props
   const { t } = useTranslation('shared')
 
   return (
     <Flex
       justifyContent={JUSTIFY_CENTER}
       flexDirection={DIRECTION_COLUMN}
       textTransform={TEXT_TRANSFORM_UPPERCASE}
-      boxShadow="0px -4px 12px rgba(0, 0, 0, 0.15)"
```
Comment (vabruzzo) [inline app/src/organisms/ConfigurePipette/ConfigFormSubmitButton.tsx:25]: we can keep the quotes here instead of `{'string'}`

Comment (shlokamin) [reply app/src/organisms/ConfigurePipette/ConfigFormSubmitButton.tsx:25]: you mean revert back to double quotes?

Comment (vabruzzo) [reply app/src/organisms/ConfigurePipette/ConfigFormSubmitButton.tsx:25]: yes we can revert to double quotes

Comment (vabruzzo) [reply app/src/organisms/ConfigurePipette/ConfigFormSubmitButton.tsx:25]: to clarify, we can revert to double quotes without the brackets

Comment (shlokamin) [reply app/src/organisms/ConfigurePipette/ConfigFormSubmitButton.tsx:25]: reverted!

**Review by vabruzzo — APPROVED** [review]
looks good

**Review by mcous — APPROVED** [review]

**Pull request merged**